curs/curs-18-19-linking/README.md

```diff
@@ -433,11 +433,240 @@ Adică locul unde era referit simbolul nedefinit va fi acum completat cu adresa
 
 ## Unificarea secțiunilor
 
-TODO
+Fișierul executabil rezultat în urma linkării conține o secțiune `.text`, o secțiune `.data` (date inițializate), o secțiune `.bss` (date neinițializate) etc.
+Toate secțiunile de același tip din cadrul fișierelor obiect linkate sunt unificate într-o singură secțiune în cadrul fișierului executabil rezultat.
+Ordinea în care secțiunile sunt agregate din diferitele fișiere obiect linkate este la latitudinea linkerului; uzual este chiar ordinea folosită în comanda de linkare.
 
-## Rezolvarea simbolurilor
+Urmărim secțiunea `.text` în fișierele obiect `one.o` și `start.o` și în fișierul executabil `one`:
+```
+[..]/01-one-file$ objdump -j .text -h one.o
 
-TODO
+one.o:     file format elf32-i386
+
+Sections:
+Idx Name          Size      VMA       LMA       File off  Algn
+  0 .text         0000002c  00000000  00000000  00000034  2**0
+                  CONTENTS, ALLOC, LOAD, RELOC, READONLY, CODE
+[..]/01-one-file$ objdump -j .text -h start.o
+
+start.o:     file format elf32-i386
+
+Sections:
+Idx Name          Size      VMA       LMA       File off  Algn
+  0 .text         0000000e  00000000  00000000  00000130  2**4
+                  CONTENTS, ALLOC, LOAD, RELOC, READONLY, CODE
+[..]/01-one-file$ objdump -j .text -h one
+
+one:     file format elf32-i386
+
+Sections:
+Idx Name          Size      VMA       LMA       File off  Algn
+  1 .text         0000003e  08048100  08048100  00000100  2**4
+                  CONTENTS, ALLOC, LOAD, READONLY, CODE
+```
+Observăm că dimensiunea secțiunii `.text` este `0x2c` pentru `one.o`, `0x0e` pentru `start.o` și `0x3e` pentru `one`.
+În mod normal, dimensiunea secțiunii pentru executabil este suma dimensiunilor secțiunilor pentru fișierele obiect linkate.
+În cazul de fața, dimensiunea este mai mare (`0x3e` > `0x2c` + `0x0e`), cel mai probabil din rațiuni de aliniere.
```

Comment:
Mentionezi de mai multe ori alinierea, dar fara sa o definesti. Cred ca ar trebui fie sa adaugi un paragraf in care explici asta in linii mari, fie sa pui un link catre o explicatie.